packages/xstate-vue/test/UseService.vue

```diff
@@ -1,26 +1,34 @@
 <template>
   <div>
-    <div data-testid="count">{{ current.context.count }}</div>
+    <div data-testid="count">{{ state.context.count }}</div>
     <button data-testid="inc" @click="send('INC')">Increase</button>
   </div>
 </template>
 
 <script lang="ts">
-import { useService } from '../src/useService';
-import { Machine, assign, Interpreter, spawn, doneInvoke, State, Service } from 'xstate';
+import { useService } from '../src';
+import {
+  Machine,
+  assign,
+  Interpreter,
+  spawn,
+  doneInvoke,
+  State,
+  Service
+} from 'xstate';
 import { watch, ref } from '@vue/composition-api';
 
 export default {
   props: ['service'],
   setup(props): { service: Service } {
-    let { current, send } = useService(props.service);
+    let { state, send } = useService(props.service);
 
     watch(() => {
```

Comment:
I think this is completely not needed (but not sure, would love to get some validation from somebody), this is already handled inside `useService`, isn't it? is using `useService` from withitn `watch` even allowed in Vue?